plugins/modules/image_info.py

```diff
@@ -5,111 +5,51 @@
 
 from __future__ import absolute_import, division, print_function
 
-from typing import Any, Optional
-
 import ansible_collections.linode.cloud.plugins.module_utils.doc_fragments.image as docs_parent
 import ansible_collections.linode.cloud.plugins.module_utils.doc_fragments.image_info as docs
-from ansible_collections.linode.cloud.plugins.module_utils.linode_common import (
-    LinodeModuleBase,
-)
-from ansible_collections.linode.cloud.plugins.module_utils.linode_docs import (
-    global_authors,
-    global_requirements,
+from ansible_collections.linode.cloud.plugins.module_utils.linode_common_info import (
+    InfoModule,
+    InfoModuleAttr,
+    InfoModuleResult,
 )
 from ansible_collections.linode.cloud.plugins.module_utils.linode_helper import (
-    filter_null_values,
-)
-from ansible_specdoc.objects import (
-    FieldType,
-    SpecDocMeta,
-    SpecField,
-    SpecReturnValue,
+    safe_find,
 )
+from ansible_specdoc.objects import FieldType
 from linode_api4 import Image
 
-spec = {
-    # Disable the default values
-    "state": SpecField(type=FieldType.string, required=False, doc_hide=True),
-    "id": SpecField(
-        type=FieldType.string,
-        description=["The ID of the image."],
-        conflicts_with=["label"],
-    ),
-    "label": SpecField(
-        type=FieldType.string,
-        description=["The label of the image."],
-        conflicts_with=["id"],
-    ),
-}
-
-SPECDOC_META = SpecDocMeta(
-    description=["Get info about a Linode Image."],
-    requirements=global_requirements,
-    author=global_authors,
-    options=spec,
+module = InfoModule(
```

Comment:
Thanks for moving this over to the new base class! I think that covers everything for TPT-2925 so feel free to self-assign it and move it to done once this has been merged